README.md

```diff
@@ -245,7 +244,11 @@ Providing an alternative way to decide which requests should be proxied; In case
 
   // Custom router function
   router: function(req) {
-      return 'http://localhost:8004';
+      return {
```

Comment:
Could keep the original example?

Basically providing both examples; One with target as a string and one with the target as an object.

Reply:
The example under this one is still the same, do you think it's enough?

```
// Asynchronous router function which returns promise
  router: async function(req) {
      const url = await doSomeIO();
      return url;
  }
```

Reply:
Think splitting the original example to a string and object example would make more sense.
It'll give new user just examples to copy/paste from.

```javascript
// Custom router function (string target)
router: function(req) {
    return 'http://localhost:8004';
}
```

```javascript
// Custom router function (target object)
router: function(req) {
    return {
      protocol: 'https:', // The : is required
      host: 'localhost',
      port: 8004
    };
}
```

The async example just indicates router accepts a Promise. Hopefully devs can figure out you can resolve the Promise with the same string/object `target`

Reply:
Good idea! I made the change.